asset/quickstart/export-assets-bigquery/main_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestMain(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	os.Setenv("GOOGLE_CLOUD_PROJECT", tc.ProjectID)
+
+	ctx := context.Background()
+	// Creates a bigquery client.
+	client, err := bigquery.NewClient(ctx, tc.ProjectID)
+	if err_bq != nil {
```

Comment:
```suggestion
	if err != nil {
```

(Please make sure the test passes locally.)

Reply:
go test -v ./..
go: downloading github.com/bmatcuk/doublestar v1.2.2
=== RUN   TestBadFiles
--- PASS: TestBadFiles (0.02s)
=== RUN   TestLicense
--- PASS: TestLicense (0.27s)
=== RUN   TestRegionTags
--- PASS: TestRegionTags (29.04s)
PASS
ok  	github.com/GoogleCloudPlatform/golang-samples	29.354s

Reply:
Looks like there are still variables with underscores in the name. https://golang.org/doc/effective_go.html#mixed-caps

Also, you need to change to this directory before running that command:

```
cd asset/quickstart/export-assets-bigquery
go test -v
```

Reply:
Thanks:
go test -v
=== RUN   TestMain
--- PASS: TestMain (65.18s)
PASS
ok  	github.com/GoogleCloudPlatform/golang-samples/asset/quickstart/export-assets-bigquery	65.275s